### PR TITLE
update guide with traffic policy

### PR DESCRIPTION
**Pull request by OfTheDelmer**
Updates the guide with initial section for the traffic policy docs

### CODE REVIEW

File: docs/k8s/user-guide.mdx [hunk 1]
```diff
@@ -465,6 +465,90 @@ spec:
                   number: <service-port>
 ```
 
+## Traffic Policy
+
+Desigining and applying just traffic policies to inbound and outbound traffic is a simplified best practice for handling edge management.
```
Comment (salilsub) [inline docs/k8s/user-guide.mdx:470]:
```suggestion
Desigining and applying just traffic policies to inbound and outbound traffic is a simplified best practice for handling edge management.
```
```suggestion
Traffic policies can be applied to inbound and and outbound traffic for simplified edge management.
```

Comment (salilsub) [inline docs/k8s/user-guide.mdx:470]:
```suggestion
Traffic policies for inbound and outbound traffic can simplify edge management.
```

File: docs/k8s/user-guide.mdx [hunk 2]
```diff
@@ -465,6 +465,90 @@ spec:
...
+
+### Design
+
+Defining a traffic policy resource with rules composed of expressions and actions allows you to influence and control traffic via the policy engine.
```
Comment (salilsub) [inline docs/k8s/user-guide.mdx:474]: This language might be a little clearer.
```suggestion
`NgrokTrafficPolicy` objects can be defined with rules composed of expressions and actions that validate and filter traffic via the policy engine.
```

File: docs/k8s/user-guide.mdx [hunk 3]
```diff
@@ -465,6 +465,90 @@ spec:
...
+              service:
+                name: <service-name>
+                port:
+                  number: <service-port>
+```
+
+With just the `k8s.ngrok.com/traffic-policy` annotation we've applied our policy to ingress.
+
+:::note
+Utilize either the module set or the traffic policy annotations on an ingress. Trying to utilize both features will result in an error.
```
Comment (salilsub) [inline docs/k8s/user-guide.mdx:545]:
```suggestion
Using both an `NgrokModuleSet` and an `NgrokTrafficPolicy` will result in an error.
```

File: docs/k8s/user-guide.mdx [hunk 4]
```diff
@@ -465,6 +465,90 @@ spec:
...
+```
+
+Learn more about the policy expressions and actions here:
+
+- [Policy Expressions](https://ngrok.com/docs/http/traffic-policy/expressions/)
+- [Policy Actions](https://ngrok.com/docs/http/traffic-policy/actions/)
+
+### Composition
+
+With a traffic policy designed for your production ingress use case you can easily add the policy and deploy with an annnotation.
```
Comment (salilsub) [inline docs/k8s/user-guide.mdx:504]:
```suggestion
Traffic policies can be added to Ingress objects using annotations.
```

File: docs/k8s/user-guide.mdx [hunk 5]
```diff
@@ -465,6 +465,90 @@ spec:
...
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: <service-name>
+                port:
+                  number: <service-port>
+```
+
+With just the `k8s.ngrok.com/traffic-policy` annotation we've applied our policy to ingress.
```
Comment (salilsub) [inline docs/k8s/user-guide.mdx:542]: I'm not sure this line adds much.
```suggestion
```

**Review by salilsub — CHANGES_REQUESTED** [review]
Made some suggestions.

File: docs/k8s/user-guide.mdx (hunk 2 above)
Comment (OfTheDelmer) [reply docs/k8s/user-guide.mdx:474]: This was tricky, I got caught up in the `influx and control traffic` phrasing for traffic policies in `docs/http/index.mdx` and the three other `docs/{http,tcp,tls}/traffic-policy/index.mdx`, but this nails it.

**Review by salilsub — APPROVED** [review]
LGTM

**Pull request merged**